contracts/mirror_gov/src/staking.rs

```diff
@@ -147,6 +141,129 @@ fn compute_locked_balance<S: Storage, A: Api, Q: Querier>(
         .unwrap_or_default())
 }
 
+pub fn deposit_reward<S: Storage, A: Api, Q: Querier>(
+    deps: &mut Extern<S, A, Q>,
+    _env: Env,
+    _sender: HumanAddr,
+    amount: Uint128,
+) -> HandleResult {
+    let config = config_read(&deps.storage).load()?;
+
+    let mut polls_in_progress = read_polls(
```

Comment:
This function has hard cap, only `30` items will be returned. maybe we can pass some flag to specify receive all the elms